posthog/session_recordings/queries/test/session_replay_sql.py

```diff
@@ -146,6 +148,19 @@ def produce_replay_summary(
         sql=INSERT_SINGLE_SESSION_REPLAY,
         data=data,
     )
+    if ensure_analytics_event_in_session:
+        # Only importing from posthog.test.base if needed
+        from posthog.test.base import _create_event, flush_persons_and_events
+
+        # It's best to also create a random analytics event, since sessions querying does a JOIN with events in
+        # any person-ID-on-events mode - sessions without an analytics event are excluded
+        _create_event(
+            distinct_id=data["distinct_id"],
+            event="foobarino",
+            properties={"$session_id": data["session_id"]},
+            team_id=team_id,
+        )
+        flush_persons_and_events()
```

Comment:
This turned out to be necessary, as the session recordings query works quite differently with `PersonsOnEventsMode.DISABLED` vs. with `PersonsOnEventsMode.PERSON_ID_OVERRIDE_PROPERTIES_JOINED`. CC @PostHog/team-replay for a sanity check

Reply:
yep... we've got the laziest implementation possible right now.

i want to figure out how we can avoid having our own events query so we can just delegate completely

i guess we need to always `select where session_id in (select $session_id from events where...)` or something but am letting future me deal with that 🤣